develop/EPM_Integration_Training/Other_Options/Other_Options.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: EpmIntegrationTrainingOtherOptions
+---
+
+# Other configuration options
+
+## Titlebox Data section
```

Comment:
This section is not clear to me. What exactly is the "Titlebox Data section" and what makes it different from the configuration of any other data page of a protocol? Also, could you explain what is shown in the top screenshot?

Reply:
If the data page is not normally shown, would it not make more sense to (also) show the end result for the user here, where the EPM entity is viewed through the topology app? That might also make it more clear what this title box data configuration is needed for, because right now, that isn't clear to me.